### DISCUSSION TITLE
No media to process in current tab

### DISCUSSION THREAD

**Discussion by mattomeara5150** [aclap-dev/video-downloadhelper#523] [Help and Q&A]
### Your question

Recently it appears that the Video DownloadHelper addon in my Edge browser updated itself and now when I try and download any videos I see the message "No media to process in the current tab. Click play on video to help detect files"
I have tried clicking on play but still see the message. Can someone please help

### Addon & CoApp version and details

version: 9.0.1.13
target: microsoft
channel: stable
lang: en-GB
coapp: "checking"
license: {"checking":true}
platform: x86-64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36 Edg/113.0.1774.57
{
    "download_directory": "/Users/matt/Downloads/[]/",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 9
}

**Comment by paulrouget** [comment]
Hmm, that's weird. Restarting Edge doesn't fix the issue?

**Comment by paulrouget** [comment]
ok - there seems to be a problem indeed (the `checking` status in your log is not normal). Can you confirm that the CoApp is not recognized and that the License is not being validated?

**Comment by paulrouget** [comment]
Do you mind trying this:

- First, tell me which version of Edge you're using
- Open Video DownloadHelper popup
- Right click inside the popup
- Click "Inspect" - a window will open
- Click on "Console" at the very top
- Tell me if it shows any errors
- Type this in the console: `chrome.runtime.lastError` and press Enter. Tell me what it says.

Thank you for your help!

**Reply by mattomeara5150** [reply]
I'm using Version 113.0.1774.57 (Official build) (x86_64)

I can see this in the "Console" tab

main.js:31  Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'setOptions')
    at sl (main.js:31:24774)
    at Bf (main.js:31:24699)
    at main.js:38:2877

running the command you supplied just returns "Undefined"

**Reply by paulrouget** [reply]
Thank you for taking the time to check for me.

The issue is that your Edge is too old. Can you please update it?

Fixed in an upcoming release: https://github.com/aclap-dev/video-downloadhelper/discussions/538

**Reply by mattomeara5150** [reply]
Done, this fixed the problem, thank you!

**Reply by donpark2000** [reply]
I found the same problem but if I change the Downloadhelper settings to the classic UI everything works as before. I'm using chromium browser (the open source one - not google chrome) on linux.

**Comment by Hirishiolo** [comment]
same problem here with Chrome [Version 109.0.5414.120 (Official Build) (64-bit)] but I cannot update the browser because it is last windows 7 support version... thank you.

**Reply by paulrouget** [reply]
Do you mind trying this:

- Open Video DownloadHelper popup
- Right click inside the popup
- Click "Inspect" - a window will open
- Click on "Console" at the very top
- Tell me if it shows any errors

**Reply by paulrouget** [reply]
I fixed it: https://github.com/aclap-dev/video-downloadhelper/discussions/538

**Comment by czarnynight** [comment]
same problem on chrome and opera gx.
